### PR TITLE
TAT-127 create shell for spa

### PR DESCRIPTION
**Pull request by allisonrobbins**
Set up scaffolding for vue app
* Forked [vue boilerplate template](https://gitlab.mitre.org/app-lab/al-boilerplate) for initial setup
* Built the four main pages for the app
* Added a nav bar

### CODE REVIEW

**Comment by mehaase** [comment]
I downloaded it and ran locally. I'll let @mikecarenzo handle the code review, but the shell looks good to me.

I also added (and assigned to Mike) a new Jira issue to set up automated previews after we merge this PR, so that its automaticaly built and deployed like Mappings Explorer is.

Edit: I did notice one issue, which is that it doesn't seem to work in a mobile layout. Are you going to add that in a future PR? Let's do that before we get too far down the road of building everything out -- it's easier to start responsive than to retrofit it.

File: .gitlab-ci.yml
Comment (mikecarenzo) [inline .gitlab-ci.yml:1]: We can remove this since we're not on GitLab.

File: .eslintrc.cjs
Comment (mikecarenzo) [inline .eslintrc.cjs]: Uninstall `prettier` and configure as eslint as necessary.

File: .vscode/settings.json
Comment (mikecarenzo) [inline .vscode/settings.json]: Remove this section.

File: .vscode/settings.json
Comment (mikecarenzo) [inline .vscode/settings.json]: Remove this section.

File: package.json
Comment (mikecarenzo) [inline package.json]: Uninstall `prettier` and configure as eslint as necessary.

**Review by mikecarenzo — APPROVED** [review]

**Comment by allisonrobbins** [comment]
@mehaase removed a few things from Mike's suggestions

Also the mobile menu is coming in the next PR

**Review by mehaase — APPROVED** [review]

**Pull request merged**
